apps/src/templates/teacherHomepage/UiTip.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const UiTip = React.createClass({
+  propTypes: {
+    index: React.PropTypes.number,
+    position: React.PropTypes.object,
+    text: React.PropTypes.string,
+    closeClicked: React.PropTypes.func.isRequired
+  },
+
+  closeClicked(index) {
+    this.props.closeClicked(index);
+  },
+
+  render() {
+    const { index, position, text } = this.props;
+
+    return (
+      <div>
+        <div
+          className="arrow_box_up"
```

Comment:
Yes, for the `:after` selector.  Wasn't sure how to do it in React.  

https://github.com/code-dot-org/code-dot-org/pull/15463/files#diff-2f5fadbda247dcceb98bc8410d0ecaa3R2880